testplan/web_ui/testing/src/AssertionPane/AssertionTypes/GraphAssertions/DiscreteChartAssertion.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import 'react-vis/dist/style.css';
+import './App.css';
+import * as GraphUtil from './graphUtils';
+import {
+  RadialChart
+} from 'react-vis';
+
+/**
+ * Component that are used to render a Chart (Data visualisations that don't require an XY axis).
+ */
+
+class DiscreteChartAssertion extends Component  {
+  components = {
+    Pie: RadialChart
+    }
+
+
+  render(){
+    let data = this.props.assertion.graph_data;
+    const graph_type = this.props.assertion.graph_type
+    const GraphComponent = this.components[graph_type];
+    let series_options = this.props.assertion.series_options;
+   // let graph_options = this.props.assertion.graph_options;
+
+    var plots = [];
```

Comment:
Generally I think it is better to use `let` instead of `var` to declare variables, due to better enforcement of scoping and re-declaration: https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var

In fact, I think that plot_options and series_colour can be declared inside the loop body as `const` values here? It's good practice to avoid mutability and use `const` wherever you can :+1: